english/talk_to_neighbour/woman/attack.md

```diff
@@ -0,0 +1 @@
+The woman deftly avoids your wild attack and steps back, adopting a defensive posture.  The man joins her side and they appear to be waiting to see what you'll do next.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.